moveit_core/collision_detection/include/moveit/collision_detection/collision_common.h

```diff
@@ -144,12 +144,7 @@ struct CostSource
 /** \brief Representation of a collision checking result */
 struct CollisionResult
 {
-  CollisionResult()
-    : collision(false)
-    , distance(std::numeric_limits<double>::max())
-    , contact_count(0)
-    , collision_log_period(5)
-    , collision_log_name("collision_common")
```

Comment:
It doesn't make sense to add these constants as extra variables to a lightweight and often used struct like `CollisionResult`.